src/status_im/contexts/wallet/router/schema.cljs

```diff
@@ -0,0 +1,86 @@
+(ns status-im.contexts.wallet.router.schema)
+
+(def ?fee-modes
+  [:enum
+   :tx-fee-mode/custom
+   :tx-fee-mode/normal
+   :tx-fee-mode/fast
+   :tx-fee-mode/urgent])
+
+(def ^:private ?hex
+  [:and :string [:re #"^0x[0-9a-fA-F]+$"]])
+
+(def ^:private ?suggested-levels-for-max-fees-per-gas
+  [:map {:closed? true}
```

Comment:
Is there a reason this map in particular is closed?

Reply:
it's a leftover, cause at the beginning I added schemas only for the router fields we use, so I closed this one since we use all the fields. Later on I decided to cover all the fields though. Also agree that on our side schemas should be open for data coming from external sources e.g. status-go

Reply:
Thanks @clauxx, makes sense to me.